src/main/resources/templates/animal/about.ftl

```diff
@@ -0,0 +1,1226 @@
+                                                    <a href="#">Які умови?</a></li>
+                                                <li id="menu-item-47"
+                                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-47">
+                                                    <a href="#">Контакти притулку</a></li>
+                                                <li id="menu-item-47"
+                                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-47">
+                                                    <a href="#">Реквізити оплати</a></li>
+                                                <li id="menu-item-47"
+                                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-47">
+                                                    <a href="#">Пошук</a></li>
+                                                <li id="menu-item-47"
+                                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-47">
+                                                    <a href="#">Партнери</a></li>
+                                                <li id="menu-item-47"
+                                                    class="menu-item menu-item-type-custom menu-item-object-custom menu-item-47">
+                                                    <a href="#">Новини</a></li>
+                                            </ul>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="column3 footer_col3">
+                                <div class="column_inner">
+                                    <div id="text-4" class="widget widget_text"><h5>Розташування притулку</h5>
+                                        <div class="textwidget">
+
+                                            <div class="vc_empty_space" style="height: 15px">
+                                                    <span
+                                                        class="vc_empty_space_inner">
+                                                        <span
+                                                            class="empty_space_image">
+                                                        </span>
+                                                        </span>
+                                                    </div>
+                                            <a href="https://www.google.com/maps/place/%D0%91%D0%9E+%D0%94%D1%96%D0%BC+%D0%A1%D1%96%D1%80%D0%BA%D0%B0+.+%D0%9F%D1%80%D0%B8%D1%82%D1%83%D0%BB%D0%BE%D0%BA/@48.8942963,24.7626555,17z/data=!3m1!4b1!4m5!3m4!1s0x4730c391910041fd:0x6a789a2223e12e2d!8m2!3d48.8942963!4d24.7648442">
```

Comment:
Make footer more informal
Delete useless and  nonactual information
Add gmap visualisation

Reply:
Done